Assets/Scripts/SS3D/Systems/PlayerControl/PlayerControlSystem.cs

```diff
@@ -23,10 +24,11 @@ public sealed class PlayerControlSystem : NetworkedSystem
         [SerializeField] private NetworkObject _unauthorizedUserPrefab;
         [SerializeField] private NetworkObject _soulPrefab;
 
-        [SyncObject]
-        private readonly SyncList<Soul> _serverSouls = new();
-        [SyncObject] 
-        private readonly SyncList<Soul> _onlineSouls = new();
+        [SyncObject] private readonly SyncList<Soul> _serverSouls = new();
+        public List<Soul> ServerSouls => _serverSouls.ToList();
```

Comment:
You can avoid copying the list here by returning the interface:
```suggestion
        public IList<Soul> ServerSouls => _serverSouls;
```